README.md

```diff
@@ -1,2 +1,35 @@
-# hw01
-Repository for first CFSS homework assignment
+# HW 01 #
```

Comment:
Coding style: NA/+ (no coding required for this assignment)
Coding strategy: NA/+
Presentation: +
Achievement, creativity: + 
Ease of access: +

All of the requirements for the assignments were met. The biography is easy to read and attractive to look at. The description of the submission process is detailed and clear.

Specifics:
- I like the emojis!
- The indents are a nice touch.